packages/datagateway-dataview/src/settings.ts

```diff
@@ -6,6 +6,7 @@ export interface DataviewSettings {
   apiUrl: string;
   downloadApiUrl: string;
   idsUrl: string;
+  icatUrl: string;
```

Comment:
Can we remove this? We have a issue to track improving the developer login process here: https://github.com/ral-facilities/datagateway/issues/1400. At the very least `icatUrl` should be optional (as it's not required in prod)